src/systemTest/java/com/flightstats/hub/system/service/CallbackService.java

```diff
@@ -92,6 +93,18 @@ public boolean areItemsEventuallySentToWebhook(String webhookName, List<String>
         }
     }
 
+    public boolean itemsNotSentToWebhook(String webhookName, List<String> channelItems, WebhookType type) {
+        try {
+            int assertAfterWaitSeconds = type.equals(WebhookType.MINUTE) ? 90 : 30;
+            Thread.sleep(assertAfterWaitSeconds * 1000);
```

Comment:
The variable name already does a good job of making it clear, but I usually try to use the TimeUnit class rather than express the math in the code. E.g:
```
Thread.sleep(TimeUnit.SECONDS(assertAfterWaitSeconds).toMillis());
```